portfolio/src/main/webapp/index.html

```diff
@@ -2,74 +2,94 @@
+              from STEP!
+            </p>
+            <a href="https://www.linkedin.com/in/shandy-kim-63242b168" target="_blank">LinkedIn</a>
+            <a href="https://www.github.com/shandyk18" target="_blank">GitHub</a>
+            <br>
+            <img src="images/me.jpg" alt="Picture of me" width="832" height="624">
+            <p>Here are fun facts about me:</p>
+            <ul>
+              <li>I am the oldest out of three kids</li>
+              <li>My first programming language was Python</li>
+              <li>My favorite game is <em>Fallout: New Vegas</em></li>
+              <li>I hate raw tomatoes with a great passion</li>
+              <li>I firmly believe there is a <em>Spongebob</em> quote for every situation</li>
+            </ul>
           </div>
-          <div class="slide">
-            <img src="images/dog2.jpg" alt="Luci with a flower" class = "dog-image">
+          <div id="projects" class="container">
+            <h3>Favorite Projects I Have Worked On</h3>
+            <ul>
+              <li><strong>xk OS:</strong> Last quarter, I took a class on operating systems. Throughout the quarter, a partner and I worked on implementing features of the barebones 
+                starter OS called xk. We implemented various features such as file creation, pipes, and copy-on-write fork. It was a painful but rewarding experience.
+              </li>
+              <li><strong>Dog Aging Project (DAP):</strong> One of the projects I enjoyed working on for the DAP was creating a SQL stored procedure to periodically deduplicate 
+                participants records.
+              </li>
+              <li><strong>Dogger:</strong> In my senior year of high school, I had the chance to take an online course on game development. The game <em>Dogger</em> was created from a two 
+                week game jam. It is a play on the classic game <em>Frogger</em>, where you play as a dog crossing the road to catch a chicken. I worked on the UI, powerups, and 
+                movement for the dog. You can play it at <a href="https://www.itch.io" target="_blank">here</a>.
+              </li>
+            </ul>
           </div>
-          <div class="slide">
-            <img src="images/dog3.jpg" alt="Luci as a puppy" class = "dog-image">
+          <div id="pet" class="container">
+            <h3>Pet: Luci</h3>
+            <p>My family has a pet dog named Luci. She's a Korean Jindo and is around 1 1/2 years old. Her favorite activities are chasing local rabbits and meeting new people.
+              One great thing about her is that she rarely barks, so no need to worry about noise complaints!
+            </p>
+            <!-- Container for the image gallery -->
+            <div class="container">
+              <div class="slide">
+                <img src="images/dog1.jpg" alt="Luci" class = "dog-image">
```

Comment:
nit: spacing for class="dog-image", same for the other two below.

Reply:
I'll be sure to change that